console/console-init/ui/src/modules/address-space/components/AddressSpaceList/AddressSpaceList.tsx

```diff
@@ -140,7 +172,7 @@ export const AddressSpaceListPage: React.FunctionComponent<AddressSpaceListPageP
   };
 
   //Download the certificate function
-  const downloadCertificate = async (data: IObjectMeta_v1_Input) => {
+  const onDownloadCertificate = async (data: IObjectMeta_v1_Input) => {
```

Comment:
Since there are queries in this file, it should be moved from components to containers folder

Reply:
i didn't separate the component in container. i will do in next level refactoring.